packages/babel-helper-builder-react-jsx-experimental/src/index.js

```diff
@@ -0,0 +1,739 @@
+import esutils from "esutils";
+import * as t from "@babel/types";
+import { addNamed, addNamespace, isModule } from "@babel/helper-module-imports";
+
+export function helper(babel, options) {
+  const FILE_NAME_VAR = "_jsxFileName";
+
+  const JSX_SOURCE_ANNOTATION_REGEX = /\*?\s*@jsxImportSource\s+([^\s]+)/;
+  const JSX_RUNTIME_ANNOTATION_REGEX = /\*?\s*@jsxRuntime\s+([^\s]+)/;
+
+  const JSX_ANNOTATION_REGEX = /\*?\s*@jsx\s+([^\s]+)/;
```

Comment:
Although `JSX_ANNOTATION_REGEX` exists in current source, but it seems to me that the `JSX_ANNOTATION_REGEX` implicitly requires an extra whitespace before the closing `*/`:
```js
JSX_ANNOTATION_REGEX.exec("/**@jsx dom*/")[1]
// returns `dom*/`
```

Reply:
It seems like `file.ast.comments`, which is what we're using, already strips out the the open and closing comment tags. So actually the comment `/**@jsx dom*/` would be parsed as `JSX_ANNOTATION_REGEX.exec("*@jsx dom")`, which is okay.